modules/serverless-create-func-kn.adoc

```diff
@@ -1,53 +1,33 @@
-// Module included in the following assemblies:
-//
-// * serverless/serverless-functions-getting-started.adoc
-
 [id="serverless-create-func-kn_{context}"]
 = Creating functions
 
 You can create a basic serverless function using the `kn` CLI.
 
-You can specify the runtime, trigger, image, and namespace as flags on the command line, or use the `-c` flag to start the interactive experience using the CLI prompt.
-
-The values provided for image and registry are persisted to the `func.yaml` file, so that subsequent invocations do not require the user to specify these again.
-
-.Example `func.yaml`
-[source,yaml]
-----
-name: example-function
-namespace: default
-runtime: node
-image: <image_from_registry>
-imageDigest: ""
-trigger: http
-builder: default
-builderMap:
-  default: quay.io/boson/faas-nodejs-builder
-envs: {}
-----
+You can specify the path, runtime, and template as flags on the command line, or use the `-c` flag to start the interactive experience using the CLI prompt.
```

Comment:
```suggestion
You can specify the path, runtime, and template as flags on the command line, or use the `-c` flag to start the interactive experience in the CLI prompt.
```

Reply:
ISG warns against "prompt" except for Windows-specific environments. 

", or use the `-c` flag to set [these things] interactively." 

IDK. :)

Reply:
updated to "to start the interactive experience in the terminal."